src/app/_client-services/polkadot_api_service.ts

```
@@ -1146,4 +1150,241 @@ export class PolkadotApiService {
+		votes: [track: BN, refIds: BN[], casting: any][],
+		referendas: [BN, any][],
+		lockPeriod: BN,
+		convictionMultipliers: number[]
+	): { endBlock: BN; locked: string; refId: BN; total: BN; track: BN }[] {
+		const locks: { endBlock: BN; locked: string; refId: BN; total: BN; track: BN }[] = [];
+
+		votes.forEach(([track, , casting]) => {
+			casting.votes.forEach(([refId, accountVote]: [BN, any]) => {
+				const referendaInfo = referendas.find(([id]) => Number(id) === Number(refId));
+				if (referendaInfo) {
+					const lockData = this.processVoteLock(refId, track, accountVote, referendaInfo[1], lockPeriod, convictionMultipliers);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)